Extensions/TileMap/JsExtension.js

```diff
@@ -343,6 +358,49 @@ const defineTileMap = function (
     .getCodeExtraInformation()
     .setFunctionName('getLayerIndex');
 
+  object
+    .addCondition(
+      'LevelIndex',
+      _('Level index'),
+      _('Compare the value of the level index.'),
+      _('the level index'),
+      '',
+      'JsPlatform/Extensions/tile_map.svg',
+      'JsPlatform/Extensions/tile_map.svg'
+    )
+    .addParameter('object', _('Tile map'), 'TileMap', false)
+    .useStandardRelationalOperatorParameters('number')
+    .getCodeExtraInformation()
+    .setFunctionName('getLevelndex');
+
+  object
+    .addAction(
+      'SetLevelIndex',
+      _('Level index'),
+      _('Set the level index of the Tilemap.'),
+      _('the level index'),
+      '',
+      'JsPlatform/Extensions/tile_map.svg',
+      'JsPlatform/Extensions/tile_map.svg'
+    )
+    .addParameter('object', _('Tile map'), 'TileMap', false)
+    .useStandardOperatorParameters('number')
+    .getCodeExtraInformation()
+    .setFunctionName('setLevelIndex')
+    .setGetter('getLevelIndex');
```

Comment:
I'm not sure users should be able to change the level with an action. LDTK files can contain all the data needed for a level but, with GDevelop, I think it's easier to place object instances with the scene editor. So, users will create a scene for each level and use the property to load the right level.

Reply:
Yeah, you are right. I did copy/paste of `LayerIndex`...

Reply:
removed